.github/scripts/compare-images-and-pdfs.py

```
@@ -0,0 +1,493 @@
+            # Get the old version (from previous commit)
+            # Construct URL for previous version
+            previous_raw_url = file_info.get('previous_filename', filename)
+            # We need to get this from the base ref
+            pr_data = self.get_pr_data()
+            base_sha = pr_data['base']['sha']
+
+            old_url = f"https://raw.githubusercontent.com/{self.repo}/{base_sha}/{filename}"
+            try:
+                old_content = self.download_file(old_url)
+                old_img = Image.open(BytesIO(old_content))
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)